scripts/gha_purge_successful_test_logs.py

```diff
@@ -42,11 +42,15 @@ def main() -> int:
         logging.error("Build dir %s does not exist or is not a directory", build_dir)
         return 1
 
-    with pathlib.Path(build_dir, 'Testing', 'Temporary', 'LastTestsFailed.log').open('rt') as f:
-        failed_tests = gha_tools.parse_last_tests_failed_log(f)
+    pathy_mcpathface = pathlib.Path(build_dir, 'Testing', 'Temporary', 'LastTestsFailed.log')
```

Comment:
pathy_mcpathface ?

Reply:
I really have to stop coding after happy hour....

Reply:
LOL